dockerfiles/execute/startup_script.sh

```diff
@@ -35,6 +35,14 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+DIFF_STARTING_BLOCK="-1"
```

Comment:
Not sure if this is the best approach for handling this, so would be totally up for changing it up if folks have ideas!

Reply:
This approach makes sense to me 👍 

Seems like someone running the binary directly could just omit that flag, but if we're always including the flag then I like having a default. My only thought would be wondering if we could use an env var instead of a flag (so that it was fully optional and didn't need to be in the dockerfiles at all), but I think that makes it harder to set a default - and potentially means we'd need to synchronize deploying this with a modification to our configured env vars